hack/dockerfile/install/containerd.installer

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+CONTAINERD_COMMIT=cfd04396dc68220d1cecbe686a6cc3aa5ce3667c # v1.0.2
+
+install_containerd() {
+	echo "Install containerd version $CONTAINERD_COMMIT"
+	git clone https://github.com/containerd/containerd.git "$GOPATH/src/github.com/containerd/containerd"
+	cd "$GOPATH/src/github.com/containerd/containerd"
```

Comment:
Should we include the optimisations in this PR? https://github.com/moby/moby/pull/35100#discussion_r156199005

Reply:
I'd rather get this in, then update it with optimizations.